### ISSUE TITLE
Doesn't has data if contains only one row.

### ISSUE THREAD

**Issue by xgrommx** [esvit/ng-table#527]
Hello @esvit 

I was faced to with a problem display one item of a collections. Here is an example regarding with this problem http://plnkr.co/edit/gFrI34pKofX7CgN806G4?p=preview. Maybe you or someone could help me with this problem.

**Comment by jeevajsb** [comment]
You are using data.length in count. That's why, you are not seeing anything in the table. I don't know the reason for that. 
Use count as 10 or if you want data.length, then use like data.length+1,

http://plnkr.co/edit/HhszXMXXd5h0OHUKGbKY?p=preview

**Comment by xgrommx** [comment]
@jeevajsb Yes, I know it, but it's not a correct behavior as for me. I think if a collection contains only one item then this one should be appeared on a table.
